include/multipass/cli/prompters.h

```diff
@@ -88,6 +90,26 @@ class NewPassphrasePrompter : public PassphrasePrompter
 
     std::string prompt(const std::string& text = "Please re-enter passphrase") const override;
 };
+
+class BridgePrompter : private DisabledCopyMove
+{
+public:
+    explicit BridgePrompter(Terminal* term) : term(term){};
+
+    ~BridgePrompter() = default;
+
+    bool bridge_prompt(std::vector<std::string>& nets_need_bridging) const;
+
+private:
+    BridgePrompter() = default;
+
+    Terminal* term;
+
+    const std::regex yes{"y|yes", std::regex::icase | std::regex::optimize};
+    const std::regex no{"n|no", std::regex::icase | std::regex::optimize};
+    const std::regex later{"l|later", std::regex::icase | std::regex::optimize};
+    const std::regex show{"s|show", std::regex::icase | std::regex::optimize};
```

Comment:
These need to be removed now due to https://github.com/canonical/multipass/pull/3173.

Reply:
Right. Removed.